src/release/lifecycle-policy.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Lifecycle policy
+group: release
+---
+
+For {{site.data.var.ece}} 2.3 and subsequent releases:
+
+-  Magento will provide quality fixes for a minor release for a minimum of 12 months from the general availability announcement date of the next minor software release.
+
+-  Magento will provide security fixes for a minor release for a minimum of 18 months from the general availability announcement date of the next minor software release.
+
+-  Magento will provide security fixes through patch releases for currently supported minor release versions only. Security fixes will not be backported to previous minor releases, nor to previous patch releases within supported minor releases.
+
+   For example, while 2.3 and 2.4 are currently supported minor releases, security fixes will be released as 2.3.X and 2.4.Y, where X and Y represent the next incremental patch release cumulative of all prior patches. Patch releases will not be released for prior minor releases that have reached end of support (for example, 2.2.Z), or as patches to prior patch releases (for example, 2.3.1.X).
```

Comment:
Suggested as this is how these patch releases appear on documentation & site. For me the 'p' adds clarity that this is a patch release.
```suggestion
   For example, while 2.3 and 2.4 are currently supported minor releases, security fixes will be released as 2.3.X and 2.4.Y, where X and Y represent the next incremental patch release cumulative of all prior patches. Patch releases will not be released for prior minor releases that have reached end of support (for example, 2.2.Z), or as patches to prior patch releases (for example, 2.3.1-pX).
```

Reply:
Thanks for the suggestion the issue with adding a -p is that then it signifies our security only line which could add confusion.

Reply:
But I would say that having something like 2.3.1.1 (as 2.3.1.X indicates) is also confusing as Magento only use 2.X.Y numbers. A 4th number segment never appears anywhere else. What do you think? If you think it's fine then I won't contest  😄

Reply:
@jeff-matthews - What do you think?

Reply:
I'm leaning towards what @coreydulimba-ad said @BarnyShergold.

Reply:
@jeff-matthews Okay- as the consensus is to leave as is, I'll agree

Reply:
Thanks @BarnyShergold!